2021-05-27 23:32:04,559 INFO    MainThread:22390 [wandb_setup.py:_flush():69] setting env: {}
2021-05-27 23:32:04,560 INFO    MainThread:22390 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-27 23:32:04,560 INFO    MainThread:22390 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_233204-2feugtn8/logs/debug.log
2021-05-27 23:32:04,560 INFO    MainThread:22390 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_233204-2feugtn8/logs/debug-internal.log
2021-05-27 23:32:04,560 INFO    MainThread:22390 [wandb_init.py:init():369] calling init triggers
2021-05-27 23:32:04,560 INFO    MainThread:22390 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 4, 'batch_size': 16, 'init_lr': 0.00013308785908840967}
2021-05-27 23:32:04,560 INFO    MainThread:22390 [wandb_init.py:init():418] starting backend
2021-05-27 23:32:04,560 INFO    MainThread:22390 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-27 23:32:04,570 INFO    MainThread:22390 [backend.py:ensure_launched():132] starting backend process...
2021-05-27 23:32:04,577 INFO    MainThread:22390 [backend.py:ensure_launched():136] started backend process with pid: 22399
2021-05-27 23:32:04,579 INFO    MainThread:22390 [wandb_init.py:init():423] backend started and connected
2021-05-27 23:32:04,580 INFO    MainThread:22390 [wandb_init.py:init():465] updated telemetry
2021-05-27 23:32:04,581 INFO    MainThread:22390 [wandb_init.py:init():484] communicating current version
2021-05-27 23:32:05,318 INFO    MainThread:22390 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-27 23:32:05,318 INFO    MainThread:22390 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-27 23:32:05,604 INFO    MainThread:22390 [wandb_init.py:init():522] starting run threads in backend
2021-05-27 23:32:05,640 INFO    MainThread:22390 [wandb_run.py:_console_start():1565] atexit reg
2021-05-27 23:32:05,641 INFO    MainThread:22390 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-27 23:32:05,641 INFO    MainThread:22390 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-27 23:32:05,644 INFO    MainThread:22390 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-27 23:32:05,644 INFO    MainThread:22390 [wandb_init.py:init():546] run started, returning control to user process
2021-05-27 23:32:08,925 INFO    MainThread:22390 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-27 23:32:08,930 INFO    MainThread:22390 [wandb_run.py:_restore():1507] restore
2021-05-27 23:32:10,938 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:32:11,164 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:32:11,622 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 3425
}

2021-05-27 23:32:12,018 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3842
  total_bytes: 3842
}

2021-05-27 23:32:12,119 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3842
  total_bytes: 3842
}

2021-05-27 23:32:12,221 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3842
  total_bytes: 3842
}

2021-05-27 23:32:12,322 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3842
  total_bytes: 3842
}

2021-05-27 23:32:12,423 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3842
  total_bytes: 3842
}

2021-05-27 23:32:12,524 INFO    MainThread:22390 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3842
  total_bytes: 3842
}

2021-05-27 23:32:13,621 INFO    MainThread:22390 [wandb_run.py:_show_files():1879] logging synced files
